pcweb/pages/customers/stories/bayesline.md

```diff
@@ -0,0 +1,106 @@
+---
+company: Bayesline
+description: "How Bayesline builds 4x faster using Reflex instead of Plotly Dash"
+domain: "https://bayesline.com/"
+founded: "New York, 2024"
+investors: "Y Combinator",
+meta: [
+  {
+    "name": "keywords",
+    "content": "
+      plotly dash,
+      python and react,
+      python and react js,
+      python react,
+      python web app,
+      python web app framework,
+      python web apps,
+      react js python,
+      react js with python,
+      react python,
+      react python websocket,
+      react with python,
+      web app framework python,
+      web app python,
+    "
+  }
+]
+---
+
+Why Bayesline chose Reflex instead of Plotly Dash
+
+## TLDR;
+
+Sebastian and his small team use Reflex to keep the full flexibility of a traditional ReactJS app **WITHOUT** needing to:
+
+* Learn frontend: React (JavaScript), NodeJS, TailwindCSS
+
+```md quote Misha
+Reflex definitely saved us from needing to hire a frontend engineer.
+```
+
+* Write boilerplate to stitch together their frontend and backend:
+
+```md quote Sebastian
+50% less code than Dash
```

Comment:
"50% less code than the same Dash app and easier to easier to read / write / maintain code compared to Dash"
Think we should somehow add this figure into this one and then we can use the spaghetti quote as well perhaps here?

Reply:
think the fact that the app is easier to maintain is critical as that was one of the main reasons they moved